src/main/resources/erlyberly/preferences.fxml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import java.lang.*?>
+<?import javafx.scene.layout.*?>
+
+<GridPane fx:id="root" hgap="5.0" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" 
+          vgap="5.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="erlyberly.PreferencesView">
+  <columnConstraints>
+    <ColumnConstraints hgrow="ALWAYS" minWidth="10.0" prefWidth="100.0" />
+    <ColumnConstraints hgrow="NEVER" minWidth="10.0" prefWidth="100.0" />
+  </columnConstraints>
+  <rowConstraints>
+    <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+    <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+    <RowConstraints minHeight="10.0" vgrow="SOMETIMES" />
+    <RowConstraints minHeight="10.0" prefHeight="30.0" vgrow="SOMETIMES" />
+  </rowConstraints>
+   <children>
+      <Label text="Target Node" GridPane.columnIndex="1" />
+      <TextField fx:id="nodeNameField" />
+      <Label text="Cookie" GridPane.columnIndex="1" GridPane.rowIndex="1" />
+      <TextField fx:id="cookieField" GridPane.rowIndex="1" />
+      <CheckBox fx:id="autoConnectField" text="Auto Connect?" GridPane.columnSpan="2" GridPane.rowIndex="2" />
+      <CheckBox fx:id="hideProcesses" text="Hide Processes on startup?" GridPane.columnSpan="2" GridPane.rowIndex="3" />
+      <CheckBox fx:id="hideModules" text="Hide Modules on startup?" GridPane.columnSpan="2" GridPane.rowIndex="4" />
+<!--      <Label text="processPaneWidth" GridPane.columnIndex="1" GridPane.rowIndex="5" />
+      <TextField fx:id="processPaneWidth" GridPane.rowIndex="5" />
+      <Label text="modulesPaneWidth" GridPane.columnIndex="1" GridPane.rowIndex="6" />
+      <TextField fx:id="modulesPaneWidth" GridPane.rowIndex="6" />-->
+      <Button fx:id="saveButton" defaultButton="true" maxHeight="-Infinity" maxWidth="1.7976931348623157E308" 
```

Comment:
It calls Store...
I merely added it so end-user's would't have to guess about whether it was saved or not.